cmd/notary-server/config.json

```diff
@@ -13,6 +13,6 @@
 	},
 	"storage": {
 		"backend": "mysql",
-		"db_url": "dockercondemo:dockercondemo@tcp(notarymysql:3306)/dockercondemo"
+		"db_url": "gordontheturtle:dockerrulez@tcp(notarymysql:3306)/development"
```

Comment:
I have no idea what Cobra is :)  

This thing will be deployed with Ansible, until infra switches to docker-compose, so changes to this file are for now immaterial